test/e2e/all_test.go

```diff
@@ -16,6 +18,11 @@ import "testing"
 // ingressconfig object and the defer logic for cleanup is still
 // runnng when the new test starts.
 func TestAll(t *testing.T) {
+	// This is required because controller-runtime expects its consumers to
+	// set a logger through log.SetLogger within 30 seconds of the program's
+	// initalization.
+	ctrlruntimelog.SetLogger(logr.New(ctrlruntimelog.NullLogSink{}))
```

Comment:
I think that the better place for this is `operator_test.go`, in a `init()` function. Because here the logger will be set at the test runtime. Some setup may take longer (`TestMain`) or some other test can be run before `TestAll` (I believe it's alphabetically ordered but not guaranteed).

Reply:
Yea that's fair, I didn't realize `TestAll` was really just a test itself, it wasn't as special as I thought. I also goofed up the imports.

Good suggestion, seems reasonable. I put it in an `init()` function in `operator_test.go`.